app/helpers/step_by_step_helper.rb

```diff
@@ -0,0 +1,5 @@
+module StepByStepHelper
```

Comment:
Can you add a test for this helper?

Reply:
There's no new functionality being added here, just a refactor - all current tests continue to pass. If I change `@step_by_step_page.id` to 'foo' (or anything else) the tests fail. That gives me confidence this works as expected.

I think adding a test would just end up shifting complexity into `/spec` instead and this wouldn't be much of a refactor at all.

Reply:
@ChrisBAshton I think that the values for step_by_step_page.id and current_user.name should be passed in. This will allow the helper method to be tested and give it more flexibility.

The benefit of adding a test is that you don't just have confidence that it works today, but that it continues to work in the future.

Calling `@step_by_step_page` directly in the helper method works at the moment, because so far every controller that uses the helper method has `@step_by_step_page` defined, but if someone creates a new controller, and uses another variable name e.g. `@step_nav` this helper method won't work for them.

Example test:

```
    it "adds a job to the draft worker" do
      allow(Services.publishing_api).to receive(:lookup_content_id)
      user = create(:user, name: "New author")
      step_by_step_page = create(:step_by_step_page)

      Sidekiq::Testing.fake! do
        expect {
          helper.update_downstream(step_by_step_page.id, user.name)
        }.to change(StepByStepDraftUpdateWorker.jobs, :size).by(1)
      end
    end
```

Calls to `update_downstream` would have to be amended for this to work: 
```
update_downstream(step_by_step_page.id, current_user.name)
```